src/main/java/sort/NaturalOrderComparator.java

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
I assumed that is done by somebody else 😉 (comment below 🙈 ). I have comments to this code but I leave it as it is